packages/app/src/notification/listeners/outbox-notification.listener.ts

```diff
@@ -24,6 +24,13 @@ const resourceTypeMap: Record<OutboxEntryType, NotificationResourceType> = {
 
 const newNotifications = (_: any, data: EntryChangesEvent[]): void => {
   data.forEach((item) => {
+    const threadUpdate =
```

Comment:
Are we still need it?

Reply:
No, with new managing of checkboxes this is unnecessary, [106999b](https://github.com/mudita/mudita-center/pull/864/commits/106999bfe9a7c55f9033a87e93120de5f3ffffd0)